sources/packages/api/src-sql/COEDeniedReason/Create-coe-denied-reason.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS coe_denied_reason(
```

Comment:
In the beginning, we were using all the tables with plural names and I believe that we lost track of it when few tables were created not following it.
I would recommend stop creating tables in singular and keep all of them from now on using the plural.

Reply:
I was following the `pir_denied_reason` name of consistency, make sense, updating

Reply:
Yes, I got it. Thanks for doing the change.
@guru-aot and @dheepak-aot, please let's ensure that we are creating new tables with names in the plural.